web/src/settings_org.js

```diff
@@ -262,21 +264,51 @@ function set_msg_move_limit_setting(property_name) {
     enable_or_disable_related_message_move_time_limit_setting(property_name, disable_setting);
 }
 
-function message_delete_limit_setting_enabled(setting_value) {
+function message_delete_limit_setting_enabled() {
```

Comment:
for this I guess you can use `user_groups.get_recursive_subgroups(realm_can_delete_any_message_group)` and check if `realm_delete_own_message_policy` in the returned set?